src/ctia/stores/es/crud.clj

```diff
@@ -448,6 +448,13 @@ It returns the documents with full hits meta data including the real index in wh
                                            ident
                                            get-in-config))))))
 
+(defn set-in-analyzer
+  [query-mode]
+  (cond-> {}
+    (contains? #{:query_string :simple_query_string} query-mode)
+    (assoc :analyzer :search_analyzer
+           :quote_analyzer :standard)))
```

Comment:
If we tune this value we must stick to[ the custom analyzer we defined](https://github.com/threatgrid/ctia/blob/7df4bc6ed216bd15a0c5c1c35bd754be290739bd/src/ctia/stores/es/mapping.clj#L282), which properly covers our needs (like properly analyzing urls for instance). 
```suggestion
           :quote_analyzer :text_analyzer)))
```
(I think it can fix the related issue, met me know)